modules/ipfs-cpinner-client/src/auth-manager/testing.ts

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios'
```

Comment:
Is "testing.ts" the right name for the file?

Reply:
when creating the cookies set them as 'secure'

Reply:
_testing_ is because this module is used only for testing. It is emulating what a browser would do with the cookies

Reply:
<img width="601" alt="Screen Shot 2021-04-22 at 23 31 55" src="https://user-images.githubusercontent.com/36084092/115809798-25058100-a3c3-11eb-8964-cac35905f6a6.png">

Cookies now have _secure_ flag. Is it ok this way?